x/auth/client/cli/tx_sign.go

```diff
@@ -129,6 +130,12 @@ func makeSignBatchCmd() func(cmd *cobra.Command, args []string) error {
 				}
 				// increment the gas
 				newGasLimit += fe.GetTx().GetGas()
+				// Individual fee values from each transaction need to be aggregated to calculate the total fee for the batch of transactions.
```

Comment:
We can split this comment over multiple lines, please. Keep it within the width of 50-60 characters per line given that it is already indented.

Reply:
Please fix this comment to
```go
// Individual fee values from each transaction need to be
// aggregated to calculate the total fees for the batch of transactions.
```